.circleci/config.yml

```diff
@@ -43,10 +43,14 @@ jobs:
           command: $GRADLEW checkstyle
       - run:
           name: ktlint
-          command: $GRADLEW ktlint
+          command: $GRADLEW ciktlint
       - run:
           name: Lint
           command: $GRADLEW lintVanillaRelease || (grep -A20 -B2 'severity="Error"' -r --include="*.xml" WordPress libs; exit 1)
+      - run:
+          name: Violations
+          when: on_fail
+          command: $GRADLEW violationCommentsToGitHub -DGITHUB_PULLREQUESTID=${CIRCLE_PULL_REQUEST##*/} -DGITHUB_OAUTH2TOKEN=$DANGER_GITHUB_API_TOKEN
```

Comment:
I have re-enabled `Only build Pull Requests` in CircleCI since this won't work if a pull request isn't present.

Reply:
Also, can we rename `$DANGER_GITHUB_API_TOKEN` to `$GITHUB_API_TOKEN` since its not used for Danger anymore?

Reply:
> I have re-enabled Only build Pull Requests in CircleCI since this won't work if a pull request isn't present.

Yeah. If a Pull Request isn't present, it's ok that it doesn't do anything. It's used only to show the results of the linter, so it makes sense to have it only on Pull Requests. On branch builds, if there's a lint error, the _lint_ job fails as usual. 

> Also, can we rename $DANGER_GITHUB_API_TOKEN to $GITHUB_API_TOKEN since its not used for Danger anymore?

Yep, makes sense.

Reply:
> Yeah. If a Pull Request isn't present, it's ok that it doesn't do anything.

It's actually a little trickier than this. If we enable `Only build Pull Requests`, branches without pull requests will not be built. But without this setting `violationCommentsToGitHub` will do nothing if a build is triggered before the PR is opened.